packages/grafana-i18n/src/rules/no-untranslated-strings/no-untranslated-strings.test.js

```diff
@@ -1,8 +1,17 @@
 import { RuleTester } from 'eslint';
 
-import noUntranslatedStrings from '../rules/no-untranslated-strings.cjs';
+import noUntranslatedStrings from './no-untranslated-strings.cjs';
 
-RuleTester.setDefaultConfig({
+const filename = 'public/app/features/some-feature/nested/SomeFile.tsx';
+
+const packageName = '@grafana/i18n';
+
+const TRANS_IMPORT = `import { Trans } from '${packageName}';`;
+const T_IMPORT = `import { t } from '${packageName}/internal';`;
+const USE_TRANSLATE_IMPORT = `import { useTranslate } from '${packageName}';`;
+const TRANS_AND_USE_TRANSLATE_IMPORT = `import { Trans, useTranslate } from '${packageName}';`;
+
+const ruleTester = new RuleTester({
   languageOptions: {
     ecmaVersion: 2018,
     sourceType: 'module',
```

Comment:
Diff looks bigger but all it was was moving the `setDefaultConfig` logic into the `new` call, as apparently the other one isn't correct according to the types (but it is needed, and was working 🤷 )